release_docs/RELEASE.txt

```diff
@@ -48,13 +48,23 @@ New Features
 
     Configuration:
     -------------
+    - Autotools and CMake target added to produce doxygen generated documentation
+
+      Autoconf option is '--enable-doxygen'
+          autotools make target is 'doxygen' and will build all doxygen targets
+      CMake confgure option is 'HDF5_BUILD_DOC'.
```

Comment:
typos - confgure above, availble below.

The '--enable-doxygen' option currently defaults to on.  Should that be mentioned here so users know to disable it if they don't want it added to the build?

Reply:
Typos fixed.  enable-doxygen just enables building doxygen, but doesn't run it.  Autotools users would run "make doxygen" to generate documentation files, or whatever the equivalent is for CMake.   Enabling it shouldn't be annoying.